hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -349,20 +349,6 @@
                  "+ext_clk_type=ExtClkLowSpeed", "+use_extclk=1", "+extclk_low_speed_sel=1"]
       reseed: 5
     }
-    // usually, uart core clock should be 24 Mhz, same as the div_4_clock, regardless using ext_clk
-    // or not.
-    // This test uses the high speed ext clk 96 Mhz, but only enable div2, so that it runs at
-    // 48Mhz, so that we know ext_clk is actually used. But this test may not work in gate-sim, as
-    // div_4_clock is too fast and it may cause some timing issue
-    {
-      name: chip_sw_uart_tx_rx_alt_clk_fast_ip_clk
```

Comment:
Shall we also remove it from the testplan?

Reply:
Good point. Removed, thanks